### DISCUSSION TITLE
SVGのアップロードに対応してほしい

### DISCUSSION THREAD

**Discussion by Robot-Inventor** [increments/qiita-discussions#380] [要望・アイデア・不具合]
現状のQiitaでは、gif、jpeg、png、tiffのアップロードに[対応](https://help.qiita.com/ja/articles/qiita-image-upload#part-fad7b978e991297c)しているとのことですが、SVGのアップロードも対応していただけると助かります。

もちろん、アップロード前にほかの形式に変換すれば問題ないのでSVGに対応していないと困るというわけではありませんが、個人的にはそのままアップロードできると便利だと感じます。

**Comment by tanaka-asami415** [comment]
@Robot-Inventor 
ご意見ありがとうございます🙏
同様のお声はいただいているので対応します！
